uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/grids/impl/BaseGridRendererHelper.java

```diff
@@ -480,6 +536,7 @@ public RenderingInformation(final Bounds bounds,
             this.floatingBlockInformation = floatingBlockInformation;
             this.minVisibleRowIndex = minVisibleRowIndex;
             this.maxVisibleRowIndex = maxVisibleRowIndex;
+            this.allRowHeights = allRowHeights;
             this.visibleRowOffsets = visibleRowOffsets;
```

Comment:
@manstis 
What's the difference between `allRowHeights` and `visibleRowOffsets` ? If the latter is just a subset of the former, can't we just save the limits of the offset without iterating twice to create two lists?

Reply:
They're not quite the same.. `allRowHeights` contains the rows' heights e.g. (as an array to make it easier to understand what I'm saying) `{20, 20, 20}` for three rows of equal height. `visibleRowOffsets` contains subsequent row's Y-offset from 0 e.g. `{0, 20, 40}` for the same three rows as shown in `allRowHeights`. The latter only contains visible rows as it's used mainly for rendering the merged grid view.